apps/api/src/app/app.module.ts

```diff
@@ -86,6 +86,7 @@ import { DisabilityLicenseClientConfig } from '@island.is/clients/disability-lic
 import { DrivingLicenseApiConfig } from '@island.is/clients/driving-license'
 import { DrivingLicenseBookClientConfig } from '@island.is/clients/driving-license-book'
 import { ElectronicRegistrationsClientConfig } from '@island.is/clients/electronic-registration-statistics'
+
```

Comment:
Unnecessary change which touches the `api` project which otherwise wouldn't be affected.